pyslackersweb/cli/sql/1.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE metadata (
```

Comment:
Due to lexographical sorting, I like to use 0-padded numbers for SQL migrations. This makes it so sorting is proper verses:

* 1.sql
* 10.sql
* 2.sql